homeassistant/components/ai_task/services.yaml

```
@@ -0,0 +1,17 @@
+generate_text:
+  fields:
+    task_name:
+      example: "home summary"
+      required: true
+      selector:
+        text:
+    instructions:
+      example: "Generate a funny notification that garage door was left open"
+      required: true
+      selector:
+        text:
+    entity_id:
+      required: false
+      selector:
+        entity:
+          domain: llm_task
```

This still has `llm_task` as the domain. Also, I think you can stick `supported_features` here? but i'm not sure I have the syntax right.

```suggestion
      selector:
        entity:
          domain: ai_task
          supported_features:
            - ai_task.AITaskEntityFeature.GENERATE_TEXT
```